Rocket.Chat/Views/Avatar/AvatarView.swift

```diff
@@ -179,13 +210,12 @@ final class AvatarView: UIView {
         imageView.animatedImage = nil
         labelInitials.text = ""
     }
-
 }
 
 // MARK: Themeable
 
 extension AvatarView {
     override func applyTheme() {
-        labelInitials?.textColor = .white
+        labelInitials.textColor = .white
     }
 }
```

Comment:
👌

Reply:
I believe you need to call super here... don't you?

Reply:
Not needed in this particular case I think, but I can do it for consistency.